common/src/api/internal/shared.rs

```diff
@@ -393,6 +471,8 @@ pub struct PortConfigV2 {
     /// Whether or not to set autonegotiation
     #[serde(default)]
     pub autoneg: bool,
+    /// LLDP configuration for this port
+    pub lldp: Option<LldpPortConfig>,
```

Comment:
Does this require a `#[serde(default)]` to be backwards compatible?

Reply:
Awesome, ignore my comments then! I went searching for details about this online before leaving that comment but could not find an answer. Thanks for the dilligence.

Reply:
Like Great White says, "My, my, my, I'm once bitten, twice shy, babe".